jetty-gcloud/jetty-gcloud-session-manager/src/main/java/org/eclipse/jetty/gcloud/session/GCloudSessionDataStore.java

```diff
@@ -455,15 +457,61 @@ public int getMaxRetries()
         return _maxRetries;
     }
 
+    public void setHost(String host)
+    {
+        _host = host;
+    }
+
+    @ManagedAttribute(value = "gcloud host", readonly = true)
+    public String getHost()
+    {
+        return _host;
+    }
+
+    public void setProjectId(String projectId)
+    {
+        _projectId = projectId;
+    }
+
+    @ManagedAttribute(value = "gcloud project Id", readonly = true)
+    public String getProjectId()
+    {
+        return _projectId;
+    }
+
     @Override
     protected void doStart() throws Exception
     {
         if (!_dsProvided)
         {
+            boolean defaultInstance = true;
```

Comment:
I'm not fond of the extra boolean for `defaultInstance`. I'm a bit on the fence for configuring passwords in here as well. Isn't there a way to provide a Datastore that is already configured for the test?

Reply:
we do not need to configure password here, the issue with configuring user/password is with infinispan :) 
anyway I removed the boolean.